dist/MapBuilder.js

```diff
@@ -55,7 +55,7 @@ var MapBuilder = function MapBuilder(props) {
     initialValues: {
       lonField: fields[0].name,
       latField: fields[0].name,
-      geomField: null,
+      geomField: '',
```

Comment:
@anuveyatsu does this work?

Reply:
@julietezekwe yes should work as empty string is a falsy value